dvc/repo/__init__.py

```diff
@@ -330,6 +326,9 @@ def graph(self, stages=None, from_directory=None):
             G_active.add_node(node, stage=stage)
 
             for dep in stage.deps:
+                if dep.path_info is None:
```

Comment:
`for dep in (d for d in stage.deps if d.path_info is not None):`

Reply:
? That doesn't look easier to read to me.